docs/tour/hello_world.mdx

```diff
@@ -29,18 +29,24 @@ use `wash` to start one now.
 
 ## Starting a Host
 
-To start a host, simply run `wash up`. This will start a host with default configurations which is perfect for this tutorial, and you can exit at any time with `CTRL+c`
+To start a host, simply run `wash up`. The default settings should work well for this tutorial. The host can be killed at any time with `CTRL+c`
 
-By default, `wash` will run the host in interactive mode. If you'd rather run the host in the background, run `wash up --detached` (or `wash up -d`, for short).
+This command does a lot behind the scenes, but most importantly it spins up 3 core processes:
+- the **wasmCloud** host which manages the execution of actors and capability providers
+- a [**NATS**](https://nats.io/) server to manage communications between application components
+- a [**wadm**](/docs/category/managed-application-deployment-wadm) process which monitors the [lattice](/docs/reference/glossary#lattice) and maintains the state of managed deployments
+
+By default, `wash` will run the host in interactive mode.
+If you'd rather run the host in the background, run `wash up --detached` (or `wash up -d`, for short).
 
 ## Generating a New Project
 
-Now that we have a host to run our application on, let's generate a new actor project. This is where our "Hello World" example begins. If you're running your host in the foreground, just switch to another terminal window to run these commands.
+Now that our wasmCloud host is running, let's generate a new actor project. This is where our "Hello World" example begins. If you're running your host in the foreground, just switch to another terminal window to run these commands.
 
 This command generates a new actor project in the `./hello` directory:
 
 ```shell
-wash new actor hello
+wash new actor -t hello hello
```

Comment:
Can we do a better job explaining what -t (template) is for here?

Reply:
Good idea, we could just go back and add the long argument for clarity